lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
```

Comment:
@aaronvb Since we'll be generating the files, we'll assume the directory matches the defaults that we generate. In other words, we'll be fixing our recommended defaults.